client/components/domain-search-v2/register-domain-step/index.jsx

```diff
@@ -1826,6 +1836,7 @@ class RegisterDomainStep extends Component {
 				wpcomSubdomainSelected={ this.props.wpcomSubdomainSelected }
 				temporaryCart={ this.props.temporaryCart }
 				domainRemovalQueue={ this.props.domainRemovalQueue }
+				onAddDomain={ this.onAddDomain }
```

Comment:
Why do we need this? Calling `cart.onAddItem` should already call this function, right?

Reply:
True. We don't need it anymore. Removed.